CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## 2.15.0-rc.0 / 2019-12-18
+
+* [CHANGE] Discovery: Removed `prometheus_sd_kubernetes_cache_*` metrics. Additionally `prometheus_sd_kubernetes_workqueue_latency_seconds` and `prometheus_sd_kubernetes_workqueue_work_duration_seconds` metrics now show correct values in seconds. #6393
+* [FEATURE] API: Added new endpoint for exposing per metric metadata `/metadata`. #6420 #6442
+* [ENHANCEMENT] TSDB: Significantly reduced memory footprint of loaded TSDB blocks. #6418 #6461
+* [ENHANCEMENT] TSDB: Significantly optimized what we buffer during compaction which should result in lower memory footprint during compaction. #6422 #6452 #6468 #6475
+* [ENHANCEMENT] TSDB: Improve replay latency. #6230
+* [ENHANCEMENT] TSDB: WAL size is now used for size based retention calculation. #5886
+* [ENHANCEMENT] Remote read: Added query grouping and range hints to the remote read request #6401
+* [ENHANCEMENT] Remote write: Added `prometheus_remote_storage_sent_bytes_total` counter per queue. #6344
+* [ENHANCEMENT] Remote write: Added config option to name remote target which is added as another label dimension to `prometheus_remote_storage_*` metrics. #6043
```

Comment:
This change also changes the `queue` label that was on many remote write metrics into `url` and `remote_name` where `remote_name` is the name or the hash of the config. Do you think that is worth calling out as a CHANGE since it could break some alerts?

Reply:
Will dive into this again :+1: